data/mods/Magiclysm/itemgroups/spellbooks.json

```diff
@@ -105,7 +106,8 @@
       [ "spell_scroll_holographic_transposition", 15 ],
       [ "spell_scroll_dark_sight", 30 ],
       [ "spell_scroll_druidic_regrowth", 20 ],
-      [ "spell_scroll_stormshaper_ionization", 40 ]
+      [ "spell_scroll_stormshaper_ionization", 40 ],
+      [ "spell_scroll_improved_knock", 35 ]
```

Comment:
is the spell tier 1 or tier 2? you added it to both. and it needs to be added to the appropriate class itemgroup

Reply:
Knock is Tier 1, as it only opens wooden doors.

Improved Knock is a separate spell that can open any door, including lab doors. That is in Tier 2.

Will add them to the class itemgroup.